src/pre_defined_classes/mathematical_program/medium/q3/TrigIdentityTest.java

```diff
@@ -0,0 +1,40 @@
+package pre_defined_classes.mathematical_program.medium.q3;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TrigIdentityTest extends BaseTest {
+    final double cos2 = 0.2500000000000001;
```

Comment:
Once you set deltas on your assertions later, you could probably have this just be `0.25` and `0.75` respectively